### PR TITLE
Over deg2 km

### PR DESCRIPTION
**Pull request by LucaDeSiena**
This is the new version of the code. Please check the examples - especially try to solve the Windows bug of the envelope function- and implement it on your dataset.

### CODE REVIEW

**Review by LucaDeSiena — COMMENTED** [review]
Just check the two comments and try pushing again.

File: bin/Murat_components.m [hunk 1]
```diff
@@ -208,7 +208,7 @@
                 rapsp(index,j)                  =...
                     (rapsp1(i+1,j) + rapsp1(i+1,j) + rapsp1(i+2,j))/3;
```
[the remark this answers is not in the document]
Comment (GeoSeisUtilities) [reply bin/Murat_components.m:209]: Checked and corrected this and other bugs. I will push the codes all together after closing the other comment.

File: bin/Murat_DDcoordinates.m [hunk 2]
```diff
@@ -0,0 +1,44 @@
+%% Recalculate grid point in decimal degrees
+function DD_coord               =   Murat_DDcoordinates(Murat)
+%%
```
[the remark this answers is not in the document]
Comment (LucaDeSiena) [reply bin/Murat_DDcoordinates.m:3]: No problem, I will edit this directly in the main branch.

**Review by LucaDeSiena — COMMENTED** [review]
Now you need to finalize your review or I cannot approve.

**Review by GeoSeisUtilities — APPROVED** [review]
This should be the review you need

**Pull request merged**